FinderKeeper/Assets/Mapbox/Examples/Scripts/RotateWithLocationProvider.cs

```diff
@@ -50,7 +50,7 @@ public class RotateWithLocationProvider : MonoBehaviour
 
 		Quaternion _targetRotation;
```

Comment:
Since was a Mapbox Example script, it should ideally be copied into our script folder, then changed from there

Reply:
Agreed. Fixed